chapters/memory-layout/introduction-to-GDB/reading/README.md

```diff
@@ -0,0 +1,90 @@
+# GNU Debugger (GDB)
+
+## Starting GDB
+
+GDB is a powerful tool for debugging programs.
+It allows you to inspect the state of a program at a certain point in its execution, set breakpoints, and step through the code, among other things.
+To start GDB, you need to run the following command:
+
+```bash
+gdb [program_name]
+```
+
+## Running the Program
+
+To run the program being debugged, there are two available commands:
+
+- `r` or `run` - this command will run the program
+- `start` - unlike `run`, this command will start the program but immediately stop after entering `main`
```

Comment:
Maybe we should tell them that `start` translates into `break main` followed by `run`